persistence-modules/redis/src/test/java/com/baeldung/DeleteEverythingInRedisIntegrationTest.java

```diff
@@ -0,0 +1,91 @@
+package com.baeldung;
```

Comment:
Let's move the code to a more specific package, e.g. com.baeldung.redis.deleteeverything

Reply:
Done as suggested.